src/pointcloud_to_laserscan_nodelet.cpp

```diff
@@ -264,11 +268,14 @@ namespace pointcloud_to_laserscan
 		for (int index=0;index<ranges_size;++index){
 			if (number_of_elements_for_index[index]==number_of_points_to_ignore){
 				output.ranges[index] = ranges[index].back();
+                if (use_inf_) output_clearing.ranges[index] = std::numeric_limits<double>::infinity();
+                else output_clearing.ranges[index] = output.range_max + 1.0;
 			}
 		}
     }
 
     pub_.publish(output);
+    pub_clearing_.publish(output_clearing);
```

Comment:
The clearing scan was there before, but not in master, I don't know where it was committed